details/operator-sdk/controllers/astraconnector_controller.go

```diff
@@ -195,19 +195,18 @@ func (r *AstraConnectorController) Reconcile(ctx context.Context, req ctrl.Reque
 		if natsSyncClientStatus.AstraClusterId != "" {
 			log.Info(fmt.Sprintf("Updating CR status, clusterID: '%s'", natsSyncClientStatus.AstraClusterId))
 		}
-		_ = r.updateAstraConnectorStatus(ctx, astraConnector, natsSyncClientStatus)
-		log.Info("assignging to status", "spec", astraConnector.Spec)
+		log.Info("assigning to status", "spec", astraConnector.Spec)
 		astraConnector.Status.ObservedSpec = astraConnector.Spec
-		err = r.Status().Update(ctx, astraConnector)
+		_ = r.updateAstraConnectorStatus(ctx, astraConnector, natsSyncClientStatus)
 		if err != nil {
 			log.Error(err, "Failed to update AstraConnector status")
 			return ctrl.Result{RequeueAfter: time.Minute * conf.Config.ErrorTimeout()}, err
 		}
-		return ctrl.Result{Requeue: false}, nil
```

Comment:
Same question, why remove these?

Reply:
I just removed the requeue part because its already the default

Reply:
In that case, would you mind making them all consistent? It looks like we explicitly set it to false in some other places

Reply:
will do